src/index.ts

```diff
@@ -1,11 +1,8 @@
+/// <reference path="./typings/semver.d.ts" />
```

Comment:
You shouldn't need this - we should include `typings/*.d.ts` in the tsconfig instead.